addons/mail/static/src/components/follower_list_menu/tests/follower_list_menu_tests.js

```diff
@@ -293,6 +294,132 @@ QUnit.test('click on remove follower', async function (assert) {
     );
 });
 
+QUnit.test('Hide "Add follower" and subtypes edition/removal buttons except own user on read only record', async function (assert) {
+    assert.expect(5);
+
+    this.data['res.partner'].records.push({ id: 100, display_name: 'Partner1'});
+    this.data['mail.followers'].records.push(
+        {
+            id: 1,
+            name: "Jean Michang",
+            is_active: true,
+            partner_id: this.data.currentPartnerId,
+            res_id: 100,
+            res_model: 'res.partner',
+            is_editable: true, // mimic logged in user's follower
+        }, {
+            id: 2,
+            name: "Eden Hazard",
+            is_active: true,
+            partner_id: 11,
+            res_id: 100,
+            res_model: 'res.partner',
+        },
```

Comment:
explicitly set `is_editable` to `false` in the other case as it's the main goal of the test